src/components/mote/skjema/FlereTidspunktSkjema.js

```diff
@@ -49,9 +62,22 @@ export const FlereTidspunktSkjema = (props) => {
     avbrytFlereAlternativ,
     antallNyeTidspunkt,
   } = props;
+  const [
+    showSameTidspunktErrorMessage,
+    setShowSameTidspunktErrorMessage,
+  ] = useState(false);
   const submit = (values) => {
```

Comment:
Burde duplikatsjekken vært på validate-funksjonen i stedet for submit?

..ellers, hadde det vært mulig å filtrere bort ugyldige tidspunkt slik at man ikke kan velge de i utgangspunktet?

Reply:
Det siste du sier stemmer, det er begrenset hvor mye jobb vi gidder å legge inn nå 👍

Jeg er enig i at validering hadde vært bedre, men det blir litt rart fordi denne valideringen gjelder både dato- og tidfeltet, så da får man samme tekst under begge. Det er sikkert mulig å få det til å fungere bra, men da hadde det tatt mye mer tid, siden jeg ikke er så godt kjent med valideringen.

Å filtrere bort ting tror jeg blir vanskelig på denne typen input, med blanding av fritekst og kalender, dessuten er det dårlig design å fjerne ting, siden veilederen da ikke får noe beskjed om hvorfor det ikke er tilgjengelig.